src/ndn/transport/face.py

```diff
@@ -100,3 +101,67 @@ def __init__(self, host: str = '', port: int = 0):
     async def open(self):
         self.reader, self.writer = await aio.open_connection(self.host, self.port)
         self.running = True
+
+
+class UdpFace(Face):
+    host: str = '127.0.0.1'
+    port: int = 6363
+
+    def __init__(self, host: str = '', port: int = 0):
```

Comment:
Declare the defaults 127.0.0.1 and 6363 on this line.
Get rid of the `if` statements.
The member declarations in above three lines are also unnecessary.